include/exadg/postprocessor/kinetic_energy_spectrum.cpp

```diff
@@ -94,7 +94,8 @@ class DealSpectrumWrapper
     s.init(dim, n_cells_1D, points_src, points_dst);
 
     std::vector<dealii::types::global_dof_index> local_cells;
-    for(auto const & cell : tria.active_cell_iterators())
+    for(auto const & cell : tria.cell_iterators())
```

Comment:
@peterrum please check this change. I switched to `cell_iterators()` since the following line writes `if(cell->is_active() ...)`

Reply:
This is correct, but the previous code could have skipped `cell->is_active()` instead. But it then boils down to what we prefer: Have the explicit check `is_active` in any case, or use a more compact notation with a `for` loop over `active_cell_iterators()`. As for the mental load, I do not see a difference since one needs to be familiar with a similar vocabulary. But since things will horribly go wrong without `active_cell_iterators` in the algorithm, I would tend to keep the version proposed here.

Reply:
Personally, I would keep `active_cell_iterators()` and remove `cell->is_active()`.

Reply:
I don't have a clear preference here. Since this code was contributed by @peterrum, I think I will follow your suggestion.

Reply:
see https://github.com/exadg/exadg/pull/216/commits/e6b491128b7b60fc4f1a34db7a4d5e48977db518